paraflow-metaserver/src/main/java/cn/edu/ruc/iir/paraflow/metaserver/utils/MetaConfig.java

```diff
@@ -33,7 +37,41 @@ public void init(String configPath) throws ConfigFileNotFoundException
     public boolean validate()
     {
         // TODO alice: validate configuration content
-        return true;
+        try {
+            //dbDriver format check
+            String dbDriver = getDBDriver();
+            String regExDriver = "^[a-zA-Z]+.[a-zA-Z]+.[a-zA-Z]+$";
+            Pattern pDriver = Pattern.compile(regExDriver);
+            Matcher mDriver = pDriver.matcher(dbDriver);
+            boolean driver = mDriver.find();
+            //dbHost format check
+            String dbHost = getDBHost();
+            String regExHost = "^[a-zA-Z]+:[a-zA-Z]+://[0-9]{1,3}.[0-9]{1,3}.[0-9]{1,3}.[0-9]{1,3}:[0-9]{1,5}(/[a-zA-Z]+)+$";
```

Comment:
host name may not contain ip address, for example:
hdfs://localhost:9000/example